Assets/MixedRealityToolkit.Examples/Demos/EyeTracking/DemoVisualizer/Scripts/LogStructureEyeGaze.cs

```diff
@@ -64,13 +64,13 @@ public override object[] GetData(string inputType, string inputStatus, EyeTracki
                 0,
 
                 // Smoothed eye gaze signal 
-                EyeTrackingProvider.IsEyeGazeValid ? EyeTrackingProvider.GazeOrigin.x : 0,
-                EyeTrackingProvider.IsEyeGazeValid ? EyeTrackingProvider.GazeOrigin.y : 0,
-                EyeTrackingProvider.IsEyeGazeValid ? EyeTrackingProvider.GazeOrigin.z : 0,
+                EyeTrackingProvider.IsGazeInputEyeBased ? EyeTrackingProvider.GazeOrigin.x : 0,
```

Comment:
The LogStructureEyeGaze.cs is used as an example for tracking visual attention. Ideally, this should be possible even if eye gaze is not used as an input.

Reply:
This change only reflects field name change from IsGazeValid to IsGazeInputEyeBased . 
IsGazeInputEyeBased still returns UseEyeTracking && IsEyeTrackingAvailable.

So I believe this should be implemented in a separate PR?

Reply:
The choice of name is confusing and reflects that whether eye gaze data is available is not the same as setting whether eye gaze is used as an input control.

Reply:
@sostel this was only a rename to fix the confusion that happened in gaze provider with overriding the IsEyeGazeValid flag but actually turning it into something else. What that flag actually did was combining if we have valid eye gaze data and eye tracking was enabled by the user as active input source. 
If the demo visualizer should work as you've described there's a bug in the implementation, this code shouldn't have worked with eye tracking not being set as input source. I'm with @vaoliva  on that one - if we want this example to be fixed let's file a follow up issue and someone should thoroughly test this when doing the changes - we shouldn't mix this into this PR :)

Reply:
Just to make sure this was clear, The Eye tracking demo scenes are all working after change.